ci/image/Dockerfile.dotnet-ubuntu204-net9-build

```diff
@@ -0,0 +1,57 @@
+# Use the official Ubuntu base image from Docker Hub
+FROM ubuntu:20.04
+
+USER root
+WORKDIR /
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# create dir for workspace
+RUN mkdir -p /home/user
+RUN chmod 777 /home/user
+
+# Update packages and install any desired dependencies
+RUN apt-get update
+RUN apt-get install -y
+RUN apt-get install -y wget
+RUN apt-get install -y apt-transport-https
+RUN apt-get install -y build-essential
+RUN apt-get install -y libpng-dev
+RUN apt-get install -y libtiff-dev
+RUN apt-get install -y libjpeg-dev
+RUN apt-get install -y libx11-dev
+RUN apt-get install -y libgl1-mesa-dev
+RUN apt-get install -y libglu1-mesa-dev
+RUN apt-get install -y gcc
+RUN wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+RUN dpkg -i packages-microsoft-prod.deb
+
+#  install basic tools
+RUN apt-get install -y git
+RUN apt-get install -y zstd
+RUN apt-get install -y jq
+
+# Install .NET SDK 9.0
+RUN apt-get update
+RUN apt-get install -y
+RUN apt-get install -y dotnet-sdk-6.0
```

Comment:
do we need to install dotnet 6,7 and 9 sdk?

Reply:
I think I could not install them but installing them does not do anything bad especially that in net8 image there was also net6 sdk installed.

Reply:
If it's not needed, I would just not install them and have a smaller image

Reply:
I removed net 6.0 but preserved net 8.0

Reply:
tests are failing without net 6.0, so I restored it